docs/image_pull.md

```diff
@@ -0,0 +1,50 @@
+# Image Pull
```

Comment:
Do we need a way to include pull secrets and/or credentials for authenticated/private repositories?